CHANGELOG.md

```diff
@@ -15,6 +15,98 @@ be exported in other repos. The issue is tracked
 [here](https://github.com/kubernetes/kubernetes/issues/48209#issuecomment-314537745).
 During the transition, you'll have to copy the helper functions to your projects. 
 
+**New features:**
+
+* Add paging support
+
+   * [https://github.com/kubernetes/kubernetes/pull/51876](https://github.com/kubernetes/kubernetes/pull/51876)
+
+* Add support for client-side spam filtering of events
+
+   * [https://github.com/kubernetes/kubernetes/pull/47367](https://github.com/kubernetes/kubernetes/pull/47367)
+
+* Support http etag and caching
+
+   * [https://github.com/kubernetes/kubernetes/pull/50404](https://github.com/kubernetes/kubernetes/pull/50404)
+
+* Extend SetHeader Requests method ito accept multiple values
+
+   * [https://github.com/kubernetes/kubernetes/pull/50639](https://github.com/kubernetes/kubernetes/pull/50639)
+
+* Add a heap to client-go
+
+   * [https://github.com/kubernetes/kubernetes/pull/49752](https://github.com/kubernetes/kubernetes/pull/49752)
+
+* Add openstack auth provider
+
+   * [https://github.com/kubernetes/kubernetes/pull/39587](https://github.com/kubernetes/kubernetes/pull/39587)
+
+* Make it possible to allow discovery errors for controllers
+
+   * [https://github.com/kubernetes/kubernetes/pull/49495](https://github.com/kubernetes/kubernetes/pull/49495)
+
+* Adds metrics for checking reflector health.
+
+   * [https://github.com/kubernetes/kubernetes/pull/48224](https://github.com/kubernetes/kubernetes/pull/48224)
+
+* Client-go now includes the leaderelection package
+
+   * [https://github.com/kubernetes/kubernetes/pull/39173](https://github.com/kubernetes/kubernetes/pull/39173)
+
+* OpenAPI downloads protobuf rather than Json to improve performance
```

Comment:
Could you move it to the "breaking change" section?

Could you also update the text to "The discovery client now fetches the protobuf encoded OpenAPI schema and returns `openapi_v2.Document`"?